libs/isograph-babel-plugin/compileTag.js

```diff
@@ -8,7 +8,8 @@ function compileTag(t, path, config) {
   if (tag.isIdentifier({ name: "iso" })) {
     // Don't do anything for iso tags
   }
-  if (tag.isIdentifier({ name: "isoFetch" })) {
+  if (tag.isIdentifier({ name: "iso" })) {
```

Comment:
You'll need to add something like `text.trim().startsWith('entrypoint')` somewhere here